packages/gatsby/src/redux/persist.js

```diff
@@ -1,11 +0,0 @@
-const v8 = require(`v8`)
-const fs = require(`fs-extra`)
-
-const file = () => `${process.cwd()}/.cache/redux.state`
-
-const readFromCache = () => v8.deserialize(fs.readFileSync(file()))
-
-const writeToCache = contents =>
-  fs.writeFileSync(file(), v8.serialize(contents))
-
-module.exports = { readFromCache, writeToCache }
```

Comment:
Can we use git move (vs delete/add) to preserve history?

Reply:
blah yeah. Stupid that git did that... I'll fix this